plugin/build.gradle

```diff
@@ -285,7 +294,7 @@ List<String> jacocoExclusions = [
         'org.opensearch.ml.profile.MLPredictRequestStats',
         'org.opensearch.ml.action.deploy.TransportDeployModelAction',
         'org.opensearch.ml.action.deploy.TransportDeployModelOnNodeAction',
-	'org.opensearch.ml.action.undeploy.TransportUndeployModelsAction',
+	    'org.opensearch.ml.action.undeploy.TransportUndeployModelsAction',
```

Comment:
unneeded whitespace.

Reply:
Removed, though idk why it still showed as a change

Reply:
Looks like an 8-space tab got switched by someone somewhere to an assumed 4-space tab + 4 spaces, but GitHub does bigger tabs.  Which is why tabs are evil. :)